src/pages/sym.mdx

```diff
@@ -0,0 +1,132 @@
+Other Unicode "White Space" characters must *not* be treated as whitespace.
+Leading or trailing whitespace must be ignored.
+
+### Line processing
+
+If any part of a line cannot be processed by the implementation, it (and possibly the whole line) shall be ignored, and a warning should be produced.
+
+A line without any tokens shall be silently ignored, to permit empty lines, even if they contain whitespace and/or a comment.
+
+A line with only one token shall be ignored, being reserved for future extensions.
+Encountering one may produce a warning.
+
+Otherwise, the line defines a single **symbol**: the first token provides its location, the second token provides its name, and further tokens (if any) may provide extra metadata.
+Extra tokens are currently not specified, but may be so in the future; so, implementations should ignore them, and may produce a warning in doing so.
+
+#### Symbols
+
+A symbol is uniquely defined by the combination of its location and its name; symbol names are case-sensitive.
+If the same symbol (identical name *and* identical location) is defined twice, one of the two definitions must be silently ignored.
+
+If two distinct symbols are defined with the same name (but different locations), neither definition shall be ignored, and this should not be an error.
+A warning may be produced, though not necessarily when loading the sym file: for example, "displaying names instead of memory addresses" accomodates this case just fine, however a "go to symbol" feature wouldn't, and a warning could be produced when the user tries to go to an ambiguous target.
+
+##### Location
+
+The location shall be of the form `<bank>:<address>`, both `<bank>` and `<address>` being case-insensitive hexadecimal numbers, possibly zero-padded, separated by a colon (`:`, U+003A).
+Typical hexadecimal prefixes (`0x`, `$`, etc.) are not allowed.
+An implementation may also allow the bank to be the case-insensitive keyword `BOOT` to specify symbols in the loaded boot ROM.
+If the bank is omitted, then the colon must be as well; the results of such lines are implementation-defined.
+
+The address must be in 16-bit range (0 to $FFFF inclusive), the bank must be in 32-bit range (0 to $FFFFFFFF inclusive); implementations may abandon processing the line if either number is larger than these limits.
+Implementations may accept bank numbers higher than $FFFFFFFF.
+Nonsensical locations (e.g. `2:9800`) may produce a warning.
+
+Two locations are considered identical if they have the same bank and the same address.
+Two distinct symbols can have identical locations; in particular, <LatestStableLink to="rgbasm.5#Unions">unions</LatestStableLink> facilitate this.
+
+Locations are strictly ordered by memory region (ROM0, ROMX, VRAM, SRAM, WRAM0, WRAMX, Echo RAM, OAM, I/O, HRAM, IE), then by bank, then by address.
```

Comment:
What does this mean in terms of requirements? What part of the specification cares about this?